src/meta-schema/meta-schema.ts

```diff
@@ -368,6 +392,16 @@ export function getMetaSchema(model: Model): GraphQLSchema {
         return model.i18n.getEnumValueLocalization(enumValue as EnumValue, getResolutionOrder(resolutionOrder, context));
     }
 
+    function flexSearchPrimarySort(type: {}):{field: Field, order: 'ASC' | 'DESC'}[]{
```

Comment:
- please run the formatter here. 
- I would call the method `getFlexSearchPrimarySort`
- can't you just use the correct function argument instead of doing a type assertion below?

Reply:
for some reason the formatter wants to break the whole file but i fixed formatting problems by hand.
I can't use the correct function argutment type because the signature needs to match `IResolvers<{}, { locale: string }>`